Remotion/ObjectBinding/Web/Res/Themes/NovaViso/HTML/BocCheckBox.css

```diff
@@ -3,38 +3,64 @@
 .bocCheckBox
 {
   margin-right: 0.5em;
+  white-space: nowrap;
+}
+
+.bocCheckBox input
+{
+  position: absolute;
+  z-index: -1;
+}
+
+.bocCheckBox img
+{
+  margin: 2px;
   vertical-align: middle;
+  height: 16px;
+  width: 16px;
```

Comment:
Maybe you could create a new variable for the width/height of the images. You could call it `--images-size` because both the height and the width are 16px.

Reply:
While it's true that many icons have a standard-size of 16x16, this isn't necessarily always true. In particular, the final size of UI elements (drop downs, checkbox, etc) is not yet determined. For icons commands, menu-items, etc, a common square size is a nood idea, though. unless overridden, 16px or, possibly, 1em will be universal. That's out-of-scope for the checkbox, though.